src/terminal/adapter/adaptDispatch.cpp

```diff
@@ -2480,8 +2480,9 @@ bool AdaptDispatch::DoITerm2Action(const std::wstring_view string)
     // This is not implemented in conhost.
     if (_api.IsConsolePty())
     {
-        // Flush the frame manually, to make sure marks end up on the right line, like the alt buffer sequence.
-        _renderer.TriggerFlush(false);
+        // As of GH#8698, when we return false in ConPTY mode, we'll
+        // automatically flush the currently buffered frame before passing
+        // through this sequence.
         return false;
```

Comment:
It seems to me that these comments are somewhat misleading, since they suggest that these three places are unique in the way they flush the current frame, when in fact it applies to everywhere we're returning false. The only thing special about these cases is that they were historically the only ones that used to work, but I don't think that information is particularly useful going forward is it?